Makefile

```diff
@@ -41,8 +44,14 @@ binary-local:
 build-container:
 	docker build ${DOCKER_BUILD_ARGS} -t "$(DOCKER_IMAGE)" .
 
+docs/%.1: docs/%.1.md
+	$(GO_MD2MAN) -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@
+
+.PHONY: docs
+docs: $(MANPAGES_MD:%.md=%)
+
 clean:
-	rm -f skopeo
+	rm -f skopeo docs/*.1
 
 install: install-binary
 	install -m 644 man1/skopeo.1 ${MANINSTALLDIR}/man1/
```

Comment:
the ${MANINSTALLDIR}/man1 needs to be installed before installing the manpage. Also, source should be docs/skopeo.1 instead of man1/skopeo.1

I was thinking we could have an install-docs target, something like this:

    install: install-binary install-docs
	    # TODO(runcom)
	    #install -m 644 completion/bash/skopeo ${BASHINSTALLDIR}/

    install-binary:
	    install -d -m 0755 ${INSTALLDIR}
	    install -m 755 skopeo ${INSTALLDIR}

    install-docs:
	    install -d -m 0755 ${MANINSTALLDIR}/man1
	    install -m 644 docs/skopeo.1 ${MANINSTALLDIR}/man1/


This works fine for rpmbuild. PTAL.